src/Components/ArticleComments.jsx

```diff
@@ -1,19 +1,31 @@
-import { useEffect, useState } from "react";
-import { getCommentsByArticleId } from "../utils/utils";
+import { useContext, useEffect, useState } from "react";
+import {
+  deleteCommentsOnArticle,
+  getCommentsByArticleId,
+} from "../utils/utils";
 import { useParams } from "react-router-dom";
+import UserContext from "../Contexts/UserContext";
 
-export default function ArticleComments({
-  commentsOnArticle,
-  setCommentsOnArticle,
-}) {
+export default function ArticleComments({ commentsOnArticle, setCommentsOnArticle }) {
   const { article_id } = useParams();
+  const { loggedInUser } = useContext(UserContext);
+  const [error, setError] = useState(null);
 
   useEffect(() => {
     getCommentsByArticleId(article_id).then(({ comments }) => {
       setCommentsOnArticle(comments);
     });
   });
 
+  const handleDeleteComment = (comment_id) => {
+    deleteCommentsOnArticle(comment_id).catch((err) => {
+      setError(err);
+    });
+  };
+  if (error) {
+    return <p>{error.message}</p>;
```

Comment:
One thing to be mindful of is that those error messages are designed for developers, you may want to sometimes opt for a more user friendly message